src/video/opengl.cpp

```diff
@@ -944,10 +955,21 @@ bool OpenGLBackend::Resize(int w, int h, bool force)
 			_glBindBuffer(GL_PIXEL_UNPACK_BUFFER, this->anim_pbo);
 			_glBufferData(GL_PIXEL_UNPACK_BUFFER, pitch * h, nullptr, GL_DYNAMIC_DRAW);
 		}
-		_glBindBuffer(GL_PIXEL_UNPACK_BUFFER, 0);
+
+		if (_glClearBufferSubData != nullptr) {
+			byte b = 0;
+			_glClearBufferSubData(GL_PIXEL_UNPACK_BUFFER, GL_R8, 0, pitch * h, GL_RED, GL_UNSIGNED_BYTE, &b);
+		} else {
+			byte *buf = (byte *)_glMapBuffer(GL_PIXEL_UNPACK_BUFFER, GL_READ_WRITE);
+			for (int i = 0; i < pitch * h; i++) {
+				*buf++ = 0;
+			}
+			_glUnmapBuffer(GL_PIXEL_UNPACK_BUFFER);
+		}
```

Comment:
Maybe an idea to make a small function for this, given they are both identical?

Reply:
Maybe, but it has to help first.

Reply:
Add a very small function.

Reply:
I have to say I was mostly refering to the `if` statement, as those are identical, not? :D